### ISSUE TITLE
Slice is not shown in the display item, only in the inspector

### ISSUE THREAD

**Issue by Ion-e** [nion-software/nionswift#1286]
As title an image, unsure what's the origin of this data item exactly, however the slices should appear as an option to be changed in the display item too?

![Image](https://github.com/user-attachments/assets/9f4ed626-7ce8-43b8-b447-9969968dc768)

**Comment by cmeyer** [comment]
This is by design, although I think it's worthwhile to question the design. Let's leave this open for more thought.

**Comment by jamesrussell216** [comment]
Yeah I think the opening question is what is that "slice" referring too? It's certainly not obvious from the context, so without knowing what the data item is meant to be I don't think I could say why it should be different from say a 3/4D item.

**Comment by Ion-e** [comment]
Somewhat related issue: https://github.com/nion-software/nion-internal/issues/172

**Comment by cmeyer** [comment]
I elevated this to the prioritized list. Need design, though.

**Comment by Ion-e** [comment]
I believe showing the range of energies will make this a much more simplified implementation.

**Comment by Brow71189** [comment]
Related issue: A sequence of 1D data will show a slider in the display panel, a 1D collection of 1D data will not. The latter should really just show the slider, because it is really just the same thing as a sequence of 1D data.

**Comment by KRLango** [comment]
Plan is to add a range slider to the software (will add issue when we have it) and then put that on the image in this case.

The tricky part of this will be identifying when the slider needs to appear as it should only be for scan synchronised spectra data.

**Comment by cmeyer** [comment]
See also this issue for info on range slider (see link in comments) and general behavior:
- https://github.com/nion-software/nionswift/issues/23

**Comment by Brow71189** [comment]
> > Related issue: A sequence of 1D data will show a slider in the display panel, a 1D collection of 1D data will not. The latter should really just show the slider, because it is really just the same thing as a sequence of 1D data.
> 
> [@Brow71189](https://github.com/Brow71189) for... completeness...
> 
> What's is the difference (programmatically, experimentally, whatever-it-is-that-causes-us-to-give-these-two-things-different-names-even-though-you-say-they-are-the-same) between a sequence of 1D data and a 1D collection of 1D data?

Experimentally, you will get a "sequence" when you set the the Acquisition Panel to "Sequence of camera frames". If you do a line scan synchronized with the camera, you will get a 1D collection of spectra.

This is exactly what I meant by this system being confusing for people, because these rules seem somewhat arbitrary if you don't know all the in's and out's of the system. 

Generally you can think of a "Sequence" being a time axis, and a "Collection" being a space axis, usually our scan. But that isn't applied consistently to all cases. If it was, a normal HAADF image should be a 2D collection of 0D data. But since that isn't supported (data needs to be at least 1D), we use 0D collection of 2D data instead.
Similarly for virtual detectors, you will get a 1D collection of 2D data, where the scan axis are the two data axis, and the different detectors the collection. This is consistent with "regular" scan detectors, but not other cases.
So this whole axis naming scheme probably needs a rework at some point.
